### PR TITLE
[#2541] hide arrows buttons when auto play enabled and add hover style in app gallery

### PR DESCRIPTION
**Pull request by metzgegu**
#2541

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

- [x] Tested by quality team

### CODE REVIEW

File: frontend_app_gallery/src/container/Gallery.jsx [hunk 1]
```diff
@@ -385,10 +385,12 @@ export class Gallery extends React.Component {
 
   onClickSlickPlay (play) {
     if (play) {
+      document.getElementsByClassName('ril__navButtons').forEach(arrow => { if (arrow.style) arrow.style.visibility = 'hidden' })
```
Comment (Skylsmoi) [inline frontend_app_gallery/src/container/Gallery.jsx]: don't inline code containing an instruction in an opened block

File: frontend_app_gallery/src/container/Gallery.jsx [hunk 2]
```diff
@@ -385,10 +385,12 @@ export class Gallery extends React.Component {
...
       this.setState({
         autoPlay: setInterval(() => this.handleClickPreviousNextPage(DIRECTION.RIGHT), 3000)
       })
     } else {
+      document.getElementsByClassName('ril__navButtons').forEach(arrow => { if (arrow.style) arrow.style.visibility = 'visible' })
```
Comment (Skylsmoi) [inline frontend_app_gallery/src/container/Gallery.jsx]: same here

File: frontend_app_gallery/src/component/Carousel.jsx [hunk 3]
```diff
@@ -61,7 +61,7 @@ export class Carousel extends React.Component {
       centerMode: false,
       initialSlide: props.fileSelected,
       swipe: false,
-      arrows: !props.disableAnimation,
+      arrows: !props.disableAnimation && !props.autoPlay,
```
Comment (Skylsmoi) [inline frontend_app_gallery/src/component/Carousel.jsx]: FYI: a good practice in variable naming is to try and keep positive naming. It's usually more readable to call you variables props.enableAnimation. Here you are negating a negative variable (!disabled), it's not unclear but it less intuitive.
This comment is for general knowledge, no changes required

File: frontend_app_gallery/src/container/Gallery.jsx [hunk 4]
```diff
@@ -537,6 +553,7 @@ export class Gallery extends React.Component {
                   onMovePrevRequest={() => { this.handleClickPreviousNextPage(DIRECTION.LEFT) }}
                   onMoveNextRequest={() => { this.handleClickPreviousNextPage(DIRECTION.RIGHT) }}
                   imagePadding={0}
+                  onAfterOpen={this.handleAfterOpenReactImageLightBox.bind(this)}
```
Comment (Skylsmoi) [inline frontend_app_gallery/src/container/Gallery.jsx]: you are not supposed to bind this manually

**Comment by PhilippeAccorsi** [comment]
Tested and its working good, but need another modification :
- [ ] Hide arrows buttons also when mouse not moving (like we have for gallery__action__button__lightbox ).

**Comment by PhilippeAccorsi** [comment]
Tested on branch and verified.

**Pull request merged**